NEWS.rst

```diff
@@ -3,6 +3,94 @@ http://twistedmatrix.com/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 22.1.0.rc1 (2022-01-26)
+===============================
+
+Features
+--------
+
+- support Python 3.10b4 (#10224)
+- Type annotations have been added to the twisted.python.fakepwd module. (#10287)
+
+
+Bugfixes
+--------
+
+- twisted.internet.defer.inlineCallbacks has an improved type annotation, to avoid typing errors when it is used on a function which returns a non-None result. (#10231)
+- revert changes to ``DelayedCall.__repr__`` and ``LoopingCall.__repr__`` (#10235)
+- restore .whl building (#10236)
```

Comment:
Some of these fragments don't seem to follow the typical (documented) style, making the news file read a bit jarringly.

Reply:
Thanks. I will update the description.